src/crawlers/browser_crawler.js

```diff
@@ -351,15 +371,16 @@ class BrowserCrawler extends BasicCrawler {
 
             if (session) session.markGood();
         } finally {
-            page.close().catch((error) => this.log.debug('Error while closing page', { error }));
+            await page.close().catch((error) => this.log.debug('Error while closing page', { error }));
```

Comment:
this one

Reply:
We don't need to wait for the page to close before proceeding.

Reply:
it's a lint rule, every promise must be awaited https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md

Reply:
![image](https://user-images.githubusercontent.com/23726914/109925393-fbcd4c00-7cc1-11eb-924e-f0ef7192dd6f.jpeg)

But according to this, what we do here is correct.